provider/resources.go

```diff
@@ -2138,6 +2139,8 @@ func Provider() tfbridge.ProviderInfo {
 			"aws_sagemaker_app_image_config":       {Tok: awsResource(sagemakerMod, "AppImageConfig")},
 			"aws_sagemaker_model_package_group":    {Tok: awsResource(sagemakerMod, "ModelPackageGroup")},
 			"aws_sagemaker_app":                    {Tok: awsResource(sagemakerMod, "App")},
+			"aws_sagemaker_workforce":              {Tok: awsResource(sagemakerMod, "WorkForce")},
+			"aws_sagemaker_workteam":               {Tok: awsResource(sagemakerMod, "WorkTeam")},
```

Comment:
It looks like Amazon spells them as `Workforce` and `Workteam`:
- https://docs.aws.amazon.com/sagemaker/latest/dg/sms-workforce-create-private-console.html
- https://docs.aws.amazon.com/sagemaker/latest/APIReference/API_Workteam.html

Reply:
Will change now!